include/alpaka/warp/WarpUniformCudaHipBuiltIn.hpp

```diff
@@ -142,6 +142,31 @@ namespace alpaka
 #    else
                     ignore_unused(warp);
                     return __ballot(predicate);
+#    endif
+                }
+            };
+
+            //#################################################################
+            template<>
+            struct Shfl<WarpUniformCudaHipBuiltIn>
+            {
+                //-------------------------------------------------------------
+                __device__ static auto shfl(warp::WarpUniformCudaHipBuiltIn const& warp, float val, int srcLane)
+                    -> float
+                {
+#    if defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
+                    return __shfl_sync(activemask(warp), val, srcLane, getSize(warp));
+#    else
+                    return __shfl(val, srcLane, getSize(warp));
+#    endif
+                }
+                //-------------------------------------------------------------
+                __device__ static auto shfl(warp::WarpUniformCudaHipBuiltIn const& warp, int val, int srcLane) -> int
```

Comment:
Here you specify an explicit return value `int`, which you did not do for the other `shfl` overload. I guess this was an oversight. The other overload returns `float`, does it? Could you add the return type please?

Reply:
I think `float` is specified there, just formatted in another way

Reply:
Oh yeah, sorry! :D